accounts/queries.py

```diff
@@ -114,15 +114,25 @@ def get_child_participation_eligibility(child, study) -> bool:
     Returns:
         bool: Return true if child is eligible based on their prior study participation
     """
+    # Need to import StudyType here due to circular import problems
+    from studies.models import StudyType
+
     must_have_participated = True
     must_not_have_participated = True
 
+    # for both must have and must not have participated, ignore responses from internal studies that are empty
     if study.must_have_participated.exists():
+
         must_have_participated_all = [
             i
             for i in study.must_have_participated.all()
-            if child.responses.filter(study__exact=i).exists()
+            if child.responses.filter(study=i)
+            .exclude(
+                study__study_type_id=StudyType.get_ember_frame_player().id, sequence=[]
+            )
```

Comment:
This might be a style choice, but why not just use `study__study_type=StudyType.get_ember_frame_player()` instead?

Reply:
Because I didn't think of it 😅 Your suggestion is better - I'll fix it!